src/models/User.ts

```diff
@@ -283,12 +283,70 @@ const userSchema = new Schema(
 
 userSchema.plugin(mongoosePaginate);
 
+/**
+ * This is an interface of the user that will be returned to the client side.
+ * The differrence between this interface and the real User Interface is that it doesn't contains password field
+ * Although this is a poor way, a better way will include implementing this Model with inclusion of password field and then using that model everywhere.
+ */
+export interface InterfaceUserToReturn {
+  _id: Types.ObjectId;
+  address: {
+    city: string;
+    countryCode: string;
+    dependentLocality: string;
+    line1: string;
+    line2: string;
+    postalCode: string;
+    sortingCode: string;
+    state: string;
+  };
+  adminApproved: boolean;
+  adminFor: PopulatedDoc<InterfaceOrganization & Document>[];
```

Comment:
Why are we creating a model for this? This is already handled in the GraphQL resolvers by adding the `password: 0` argument to the database calls so that the hashed password is not fetched from the database by Mongoose.